### PR TITLE
r.in.srtm: Fix exception handling and add specific error types

### PR DESCRIPTION
**Pull request by arohanajit**
Following Exceptions added
- `zipfile.BadFIle` and `zipfile.LArgeFile` for file related errors. and `PermissionError` incase it does not have enough permission to be accessed
- In second, since `gs` module is used, I added `CalledModuleError`, I can also do a generic `Exception` here if required

### CODE REVIEW

**Comment by nilason** [comment]
I don’t understand what is “temporal” with this PR?

**Comment by arohanajit** [comment]
Yes, it shouldve been under r.*. I misclassified it

**Review by echoix — APPROVED** [review]
I went through the docs of zip file and the chosen exceptions make sense. ValueError shouldn't happen as we set the ".zip" extension. I didn't find PermissionError in the page, but it makes sense when opening a file.

So everything is fine with me!

File: scripts/r.in.srtm/r.in.srtm.py [hunk 1]
```diff
@@ -277,7 +277,7 @@ def main():
 
     try:
         gs.run_command("r.in.gdal", input=bilfile, out=tileout)
-    except:
+    except gs.CalledModuleError:
```
Comment (petrasovaa) [inline scripts/r.in.srtm/r.in.srtm.py:280]: Although this seems to work, it should be imported like that:
```
from grass.exceptions import CalledModuleError
```

**Review by petrasovaa — APPROVED** [review]

**Pull request merged**